ai_agents/travel_planner_agent/agent.py

```
@@ -0,0 +1,309 @@
+    if extras:
+        lines += ["", "**Trip-length tips**", *[f"- {item}" for item in extras]]
+
+    lines += [
+        "",
+        "**Toiletries**",
+        "- Toothbrush, toothpaste, deodorant",
+        "- Skincare and any personal care items",
+        "",
+        f"_Based on weather context: {weather_summary[:200]}..._"
+        if len(weather_summary) > 200
+        else f"_Based on weather context provided._",
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)